.github/workflows/ci.yaml

```diff
@@ -400,8 +404,81 @@ jobs:
         env:
           POSTGRES_VERSION: "13"
           TS_DEBUG_DISCO: "true"
+        shell: bash
         run: |
-          make test-postgres
+          # if macOS, install google-chrome for scaletests
+          # As another concern, should we really have this kind of external dependency
+          # requirement on standard CI?
+          if [ "${{ matrix.os }}" == "macos-latest" ]; then
+            brew install google-chrome
+          fi
+
+          # By default Go will use the number of logical CPUs, which
+          # is a fine default.
+          PARALLEL_FLAG=""
+
```

Comment:
I think most people expect this so it seems weird to set it to nothing when it's already unset. If it makes sense to you though I don't mind keeping it

Reply:
In that case it's totally fine then 👍